sashimi-webapp/src/logic/documentPackager/markdownProcessor.js

```diff
@@ -21,6 +21,7 @@ import mdAsciiMath from './markdown-it-asciimath';
 import mdDiagrams from './markdown-it-diagram';
 // Custom conditional plugin
 import mdConditional from './conditionalProcessor';
+import mdLineNumber from './plugins/markdown-it-line-number';
```

Comment:
yeah, that should be the preferred way, grouping all the self-written plugins into the plugins folder
- conditionalProcessor.js
- markdown-it-asciimath.js
- markdown-it-diagram.js